examples/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
     project(zenohpico_examples LANGUAGES C)
     include(../cmake/helpers.cmake)
     set_default_build_type(Release)
-    configure_include_project(ZENOHPICO zenohpico zenohpico::static ".." zenohpico "https://github.com/eclipse-zenoh/zenoh-pico" "")
+    configure_include_project(ZENOHPICO zenohpico zenohpico ".." zenohpico "https://github.com/eclipse-zenoh/zenoh-pico" "")
```

Comment:
This actually fixes the build if building the examples as the root project. When including the project using `add_subdirectory` or `FetchContent` the library target will be named `zenohpico`. 
The only small problem is that the target name for a `find_packaged` library will be `zenohpico::zenohpico` and this macro does not take the target namespace into account.